### ISSUE TITLE
Problem with xvfb-run inside the container

### ISSUE THREAD

**Issue by JKereliuk** [w3c/wptdashboard#266]
Right now it doesn't look like xbfb is installed by default within the container, (unless its done by some make rule I am missing).

We can fix this by adding ```apt-get install xvfb``` to a Dockerfile; however, once doing this, I try to run
```
docker exec -it -u $(id -u $USER):$(id -g $USER) wptd-dev-instance     run/run.py firefox-56.0-linux --path battery-status
```

And I get ```xvfb-run: error: xauth command not found```

Not sure how to proceed.

**Comment by jeffcarp** [comment]
`xvfb` should be installed: https://github.com/w3c/wptdashboard/blob/master/Dockerfile.jenkins#L12

**Comment by JKereliuk** [comment]
I was using the dev container, but also this is fixed by #268. Closing.
